spec/features/export_profile_feature_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'rails_helper'
 require 'capybara/rspec'
+require 'csv'
```

Comment:
This came up in testing this file in isolation, must have been `require` pollution where it was pulled in elsewhere.